.github/workflows/stateless-tests-cluster.yml

```diff
@@ -22,4 +22,4 @@ jobs:
           scripts/ci/fusequery-cluster-3-nodes.sh
           echo "Starting fuse-test"
           cd tests
-          ./fuse-test
+          ./fuse-test '^0[^4]_'
```

Comment:
Sundy, what does this argument mean?

Reply:
Explain queries are `04_xxx.sql`, so it can skip testing those queries in cluster tests.

Reply:
Thanks, make sensy